acceptance/testdata/builder/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:bionic
+
+ARG cnb_uid=1234
+ARG cnb_gid=1000
+
+ENV CNB_ASSETS=/some/asset/path
+
```

Comment:
We've put a pause on asset packages for now, so I think this could be removed.

Reply:
fixed